### DISCUSSION TITLE
Add "peek adventurer" widget

### DISCUSSION THREAD

**Discussion by ritcheedawson** [runelite/runelite#15017] [Q&A] [answered]
When I'm at Cerberus or Kraken I always have to peek to see if an adventurer is inside. it's not that big of a deal but it would be cool if there was a widget I could turn on so when I'm at those kinds of places I would already know if someone is in the room without having to use the peek feature.

**Comment by Enriath** [comment, marked as answer]
The client has no way of knowing whether there are players in peekable areas unless you use the peek option.
